### ISSUE TITLE
Can xlwings be used via excel to call an online python interpreter ?

### ISSUE THREAD

**Issue by gdv820** [ZoomerAnalytics/xlwings#597]
Hi,
I have machines where I cannot install python, but would like the user of that machine to use an excel workbook that uses xlwings and python. Is it possible for the xlwings macro, to start an online python interpreter and perform the required calculations through the RunPython macro to manipulate the excel workbook ??

**Comment by rbn920** [comment]
Your best bet I would think is to freeze your python code and use the `RunFrozenPython` function. The Fibonacci standalone example demonstrates this.

**Comment by fzumstein** [comment]
Yeah, xlwings can only deal with a locally installed Python interpreter, wether frozen or not.

**Comment by gdv820** [comment]
Alright. I did try to implement the executable via RunFrozenPython, but it tends to be a lil slow. Anyway I appreciate you guys getting back to me. If you have any thoughts on how to speed up the .exe created via python in general, let me know.

**Comment by rbn920** [comment]
I have the same problem with some of the larger spreadsheets I have created. I have tried to minimize the amount of interactions between python and excel ie, grab all the data at once, send it to python and process it, then package it up as a large list and write it back to excel in one write. That helps but after testing and tweaking it seems that the big slowdown is the initial startup time when running a frozen app, which isn't really xlwings' problem. Unfortunately I haven't been able to come up with any good work arounds.
